src/server/search/doc_accessors.cc

```diff
@@ -38,31 +38,44 @@ string_view SdsToSafeSv(sds str) {
   return str != nullptr ? string_view{str, sdslen(str)} : ""sv;
 }
 
-string PrintField(search::SchemaField::FieldType type, string_view value) {
+search::SortableValue FieldToSortableValue(search::SchemaField::FieldType type, string_view value) {
+  if (type == search::SchemaField::NUMERIC) {
+    double value_as_double = 0;
+    if (!absl::SimpleAtod(value, &value_as_double)) {  // temporary convert to double
+      VLOG(2) << "Failed to convert " << value << " to double";
```

Comment:
can it happen somehow?

Reply:
For now no, it's not possible

Reply:
then use `LOG(DFATAL)` if it can not happen with any user input due to parser checks beforehand.